lib/3scale/backend/transactor/usage_report.rb

```diff
@@ -167,7 +167,7 @@ def compute_current_value
               # this is an auth/authrep request and therefore we should sum the usage
               computed_usage = Usage.get_from this_usage, current_value
               # children can alter the resulting current value
-              add_children_usage(usage, computed_usage)
+              add_descendants_usage(usage, computed_usage)
```

Comment:
Is `add_children_usage` referenced still in `compute_usage` above still needed?

Reply:
That part of the code is only used to calculate the limits headers and I wanted to address that in a different PR.

Reply:
Not sure what you mean.
Before this PR, auth and reports calls were not taking into account the whole hierarchy. Same for the headers. My idea was to address those 2 in separate PRs.

Reply:
Ok :+1: